python/cudf/cudf/_libxx/quantiles.pyx

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from cudf._libxx.lib cimport *
+from cudf._libxx.column cimport Column
+from cudf._libxx.table cimport Table
+cimport cudf._libxx.includes.quantiles as cpp_quantiles
+
+def quantiles(Table source_table,
+              vector[double] q,
+              object interp,
+              object is_input_sorted,
+              list column_order,
+              list null_precedence):
+    cdef table_view c_input = source_table.data_view()
+    cdef vector[double] c_q = q
+    cdef interpolation c_interp = <interpolation>(<underlying_type_t_interpolation> interp)
+    cdef sorted c_is_input_sorted = <sorted>(<underlying_type_t_sorted> is_input_sorted)
+    cdef vector[order] c_column_order = vector[order](len(column_order))
```

Comment:
This initializes the vector with the default constructor of `order`, instead we'd want to do:
```
cdef vector[order] c_column_order
c_column_order.reserve(len(column_order))
```